hedera-node/src/main/java/com/hedera/services/store/contracts/ContractsStore.java

```diff
@@ -164,17 +167,26 @@ public void clearStorage(Address address) {
 
 	@Override
 	public void commit() {
-		provisionalAccountUpdates.forEach((address, evmAccount) -> {
-			final var accId = EntityIdUtils.accountParsedFromSolidityAddress(address.toArray());
-			if (ledger.exists(accId)) {
-				final var account = ledger.get(accId);
-				ledger.adjustBalance(accId, evmAccount.getBalance().toLong() - account.getBalance());
-			} else {
-				var pair = this.provisionalAccountCreations.get(accId);
-				ledger.create(pair.getKey(), accId, evmAccount.getBalance().toLong(), pair.getValue());
+		provisionalAccountCreations.forEach((accountId, entry) -> {
```

Comment:
What if the account is not a smart contract? Are we going to silently do nothing? I think that we should not